frontend/src/scenes/scenes.ts

```diff
@@ -588,6 +588,7 @@ export const routes: Record<string, [Scene | string, string]> = {
     [urls.annotation(':id')]: [Scene.DataManagement, 'annotation'],
     [urls.comments()]: [Scene.DataManagement, 'comments'],
     [urls.projectHomepage()]: [Scene.ProjectHomepage, 'projectHomepage'],
+    [urls.maxHistory()]: [Scene.Max, 'max'],
     [urls.max()]: [Scene.Max, 'max'],
```

Comment:
This seems redundant?

Reply:
- `urls.max()` --> `/max`
- `urs.maxHistory()` --> `/max/history`

Seems like a different route to me.

It should say `maxHistory` probably as the 2nd element in the array though.

Reply:
Yup